tests/testfiles/lunamark-markdown/tex-math-single-and-double-backslashes.test

```diff
@@ -0,0 +1,60 @@
+\def\markdownOptionTexMathDoubleBackslash{true}
+\def\markdownOptionTexMathSingleBackslash{true}
+\let\hat=\relax
+\let\left=\relax
+\let\xi=\relax
+\let\right=\relax
+\let\int=\relax
+\let\infty=\relax
+\let\pi=\relax
+\let\\=\relax
+<<<
+This test ensures that the Lua `texMathDoubleBackslash` and `texMathSingleBackslash` options correctly propagate
+through the plain TeX interface.
+
+\(a\)
+
+\[a\]
+
+\(a\\)
+
+\[a\\]
+
+\\(a\)
+
+\\[a\]
+
+\\(a\\)
+
+\\[a\\]
+
+\\\(a\\\)
+
+\\\[a\\\]
+>>>
+documentBegin
+codeSpan: texMathDoubleBackslash
+codeSpan: texMathSingleBackslash
+interblockSeparator
+inlineMath: a
+interblockSeparator
+displayMath: a
+interblockSeparator
+backslash
+interblockSeparator
+backslash
+interblockSeparator
+backslash
+interblockSeparator
+backslash
+interblockSeparator
+inlineMath: a
+interblockSeparator
+displayMath: a
+interblockSeparator
+backslash
+inlineMath: a\\
+interblockSeparator
+backslash
+displayMath: a\\
```

Comment:
This is incongruent with the output of Pandoc:

```
$ pandoc -f markdown+tex_math_double_backslash <<< '\\\(a\\\)'
<p>\(a\)</p>
$ pandoc -f markdown+tex_math_double_backslash <<< '\\\[a\\\]'
<p>\[a\]</p>
```

Reply:
Both extensions are enabled in this unit-test.

```
$ pandoc -f markdown+tex_math_single_backslash+tex_math_double_backslash <<< '\\\(a\\\)'
<p>\<span class="math inline">$a\\$</span></p>
```

or more importantly `tex_math_single_backslash` is enabled.

```
pandoc -f markdown+tex_math_single_backslash <<< '\\\(a\\\)'
<p>\<span class="math inline">$a\\$</span></p>
```

Reply:
Interesting. Perhaps we should add `\\\(a\\\)` and `\\\[a\\\]` to `tex-math-single-backslash.test` and `tex-math-double-backslash.test` as well? Seems like an edge case that you can imagine would parse differently in any combination of the options.